src/Integration/LocalServices/ISolutionBindingInformationProvider.cs

```diff
@@ -50,16 +50,5 @@ internal interface ISolutionBindingInformationProvider : ILocalService
         /// <seealso cref="IProjectSystemHelper.GetFilteredSolutionProjects"/></remarks>
         /// <returns>Will always return an instance, never a null</returns>
         IEnumerable<Project> GetUnboundProjects();
-
-        /// <summary>
-        /// Returns whether the solution is bound to SonarQube
-        /// </summary>
-        bool IsSolutionBound();
-
-        /// <summary>
-        /// Returns current solution project key (can be null)
-        /// </summary>
-        /// <returns></returns>
-        string GetProjectKey();
```

Comment:
The same data is available through the _IConfigurationProvider_ interface so I removed them from this interface.